coordinator/controller.go

```diff
@@ -530,11 +528,5 @@ func (c *Controller) collectMetrics() {
 		metrics.ChangefeedStateGauge.WithLabelValues("Working").Set(float64(working))
 		metrics.ChangefeedStateGauge.WithLabelValues("Stopped").Set(float64(stopped))
 		c.lastPrintStatusTime = time.Now()
-		log.Info("coordinator status",
-			zap.Int("total", total),
-			zap.Int("stopped", stopped),
-			zap.Int("absent", absent),
-			zap.Int("scheduling", scheduling),
```

Comment:
Please add the scheduling and total metrics.

Reply:
yes, I will post another PR to add metrics